src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -83,13 +85,43 @@ const GroupsView = View.extend({
   },
 });
 
+const EngagementStatusPreloadView = View.extend({
+  ui: {
+    loading: '.js-loading',
+  },
+  template: hbs`
+    <div class="js-loading">{{ @intl.patients.patient.sidebar.sidebarViews.engagementStatusPreloadView.loading }}</div>
+  `,
+  onRender() {
+    anime({
+      targets: this.ui.loading[0],
```

Comment:
sorry for the late comment.. _but_ would this work with `this.$el` not needing the `.js-loading` wrapper?